common/azure/src/main/kotlin/com/cosmotech/api/azure/adx/AzureDataExplorerClient.kt

```diff
@@ -82,6 +83,29 @@ class AzureDataExplorerClient(private val csmPlatformProperties: CsmPlatformProp
         }
     return healthBuilder.withDetail("baseUri", baseUri).build()
   }
+
+  public fun deleteDataFromScenarioRunId(
+      organizationId: String,
+      workspaceKey: String,
+      scenarioRunId: String
+  ) {
+    var databaseName = getDatabaseName(organizationId, workspaceKey)
+    try {
+      this.kustoClient.executeToJsonResult(
+          databaseName,
+          """
+        .drop extents <|
+            .show database databaseName extents
```

Comment:
You meant `${databaseName}`, right? Also I guess you should take care of properly escaping such variable in the Kusto query.